editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid

```diff
@@ -32,42 +32,131 @@ The ''action-createtiddler'' widget is invisible.
 <<.from-version "5.2.0">> The content of the <<.wid "action-createtiddler">> widget is executed after the new tiddler has been created. The title of the newly created tiddler is stored in the variable <<.var "createTiddler-title">>.
 
 |!Variables |!Description |
-|`createTiddler-title` |The tittle of the tiddler that has been created. See [[ActionCreateTiddlerWidget Example 5]] |
+|`createTiddler-title` |The tittle of the tiddler that has been created. See the 5th example |
 |`createTiddler-draftTitle` |This variable only exists to have feature parity with the deprecated parameters. It contains the title of a "draft tiddler" |
 
 ! Examples
 
 <<<
-<$macrocall $name='wikitext-example-without-html' src={{ActionCreateTiddlerWidget Example}}/>
+1) This example will create a ''new tiddler'' every time the button is clicked.
```

Comment:
Using hardcoded list numbering is not a practice I would want to recommend. If the worst comes to the worst, it would be better to use HTML syntax for `<ol>/<li>`.

Reply:
With examples we do have a lot of hardcoded numbering. eg: https://github.com/Jermolene/TiddlyWiki5/pull/7426/files#diff-ae90cbf3713502ed3ebecc9f345a9e5c95dbb06986e618697daa323025611ead from some docs recently merged.